File: DigitaliaBackendProject/Core/Models/Entities/Surveys.cs
using Dapper.Contrib.Extensions;

namespace Models.Entities
{
    public class Surveys
    {
        [Key]
        public int SurveyID { get; set; }
        public int UserID { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Question { get; set; }
        public DateTime? CreationDate { get; set; }
        public bool? IsActive { get; set; }
    }
}

File: DigitaliaFrontendProject/digitalia-votes/src/app/admin/surveys/all-surveys/surveys.ts
export interface Survey {
    surveyID: number;
    userID: number;
    title: string;
    description: string;
    question: string;
    creationDate?: string | null; 
    isActive?: boolean | null;
  }
  export interface Vote {
    voteID: number;
    surveyID: number;
    optionID: number;
    userID: number;
    voteDate?: Date; 
  }
  
  export interface validateVote{
    userID:number;
    surveyID:number;
  }